templates/company/about.html

```diff
@@ -0,0 +1,464 @@
+      </div>
+    </div>
+    <div class="row p-section--shallow">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row p-section--shallow">
+          <div class="col-6">
+            <em>"Canonical works with us as a partner and they own problems alongside us. I feel like we work as a team rather than a simple business transaction."</em>
+          </div>
+          <div class="col-3">
+            <span>
+              <strong>Curtis Haslam</strong>
+            </span>
+            <br />
+            <span class="u-text--muted">Senior Manager at Network Cloud, BT Group</span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row p-section--shallow">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row p-section--shallow">
+          <div class="col-6">
+            <em>"The customer support has been exceptional. Both through the customer success team and the customer support team."</em>
+          </div>
+          <div class="col-3">
+            <span>
+              <strong>Joseph Sinfield</strong>
+            </span>
+            <br />
+            <span class="u-text--muted">Senior Developer, Rullion</span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row p-section--shallow">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule--muted" />
+        <div class="row p-section--shallow">
+          <div class="col-6">
+            <em>"The quality of Canonical’s products has been superb and helped our teams to deliver solutions that meet our customer expectations."</em>
+          </div>
+          <div class="col-3">
+            <span>
+              <strong>Jim Sivers</strong>
+            </span>
+            <br />
+            <span class="u-text--muted">Software Development Engineering Manager, Wachter</span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <div class="p-cta-block">
+          <a href="https://ubuntu.com/engage?resource=case+study">Browse customer stories&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Where to find us</h2>
+      </div>
+      <div class="col">
+        <p>
+          With corporate hubs and a global team across APAC, EMEA and North America, we deliver the reliable support you need to innovate with confidence.
+        </p>
+        <div class="p-cta-block">
+          <a href="/contact-us">Contact our team&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+    <!-- map goes here -->
+    <div class="row">
```

Comment:
Because of the way the map is centered, only the US and UK offices show. Could we recenter so that it gives a more accurate representation of where our offices are? eg: 
![image](https://github.com/user-attachments/assets/f3268247-ac2d-4061-b2e5-7bfb0785fe53)